src/components/Attributes/SwatchRow.tsx

```diff
@@ -4,14 +4,13 @@ import {
   getErrorMessage,
   getSingleDisplayValue,
 } from "@dashboard/components/Attributes/utils";
-import HorizontalSpacer from "@dashboard/components/HorizontalSpacer";
-import SingleAutocompleteSelectField from "@dashboard/components/SingleAutocompleteSelectField";
 import { getBySlug } from "@dashboard/misc";
-import { InputAdornment } from "@material-ui/core";
+import { Box, DynamicCombobox } from "@saleor/macaw-ui/next";
+import debounce from "lodash/debounce";
```

Comment:
suggestion: `lodash/debounce` is a new import which has never been used before. Can we use `useDebounce` instead?